src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8032,6 +8022,223 @@ public static void F(IExceptionalInterface i)
             compilation2.VerifyDiagnostics(expected);
         }
 
+        private const string ExperimentalAttributeSource =
+@"using System;
+namespace Windows.Foundation.Metadata
+{
+    [AttributeUsage(
+        AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Interface | AttributeTargets.Delegate,
+        AllowMultiple = true)]
```

Comment:
The actual attribute has `AllowMultiple = false`. Updated the declaration here. However, `DeprecatedAttribute` has `AllowMultiple = true` so added a test for that case.